PrebidMobile/PrebidMobile-core/src/test/java/org/prebid/mobile/JsonUserDataTest.java

```diff
@@ -27,7 +27,7 @@ public void putUsersDataListToRequestParams_addElementsToOutsideList_elementsAdd
 
         AdUnitConfiguration configuration = new AdUnitConfiguration();
         configuration.setConfigId("configId");
-        configuration.setAdType(AdType.BANNER);
+        configuration.setAdUnitIdentifierType(AdUnitConfiguration.AdUnitIdentifierType.BANNER);
```

Comment:
This change looks not super accurate. 

```
configuration.setAdType(AdType.BANNER);
```

Looks better than 

```
configuration.setAdUnitIdentifierType(AdUnitConfiguration.AdUnitIdentifierType.BANNER);
```

My suggestions here are:

- Rename the `AdUnitIdentifierType` to `AdFormat`.
- Make the `AdFormat` a global type like the `AdType` was before.
- Rename the setter and getter from `AdUnitIdentifierType` to `AdFormat`

Reply:
Please, do not apply these changes to the current PR. We'll do it later in the scope of introducing the rendering controls. There will be a lot of work related to the AdUnitConfiguration